seeder/postgres_seeder/data_generator/seed_data/sample.csv

```diff
@@ -0,0 +1,76 @@
+product_id, style_id, size, quantity
```

Comment:
**Annotation:** This is just an example file for anyone else who gets this repository.